terragrunt/aws/api/sentinel.tf

```diff
@@ -0,0 +1,14 @@
+module "sentinel_forwarder" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v5.0.1//sentinel_forwarder"
+  function_name     = var.product_name
+  billing_tag_value = var.billing_code
+
+  layer_arn = "arn:aws:lambda:ca-central-1:283582579564:layer:aws-sentinel-connector-layer:42"
+
+  customer_id = var.log_analytics_workspace_id
+  shared_key  = var.log_analytics_workspace_key
+
```

Comment:
you might need a `log_type` just to properly namespace the logs, otherwise they will be `ApplicationLog_CL`